src/createConfirmation.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const createConfirmation = (Component, unmountDelay = 1000) => {
+const createConfirmation = (Component, unmountDelay = 1000, mountingNode = document.body) => {
```

Comment:
👋I think this might have broken server-side rendering.

On installing `0.1.17` we got the error `ReferenceError: document is not defined`, due to the way `document` is being referenced.